internal/bot/bot.go

```diff
@@ -12,12 +12,13 @@ import (
 	"time"
 
 	"github.com/google/go-github/v18/github"
-	"github.com/zendesk/term-check/internal/config"
-	gh "github.com/zendesk/term-check/pkg/github"
-	"github.com/zendesk/term-check/pkg/lib"
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
+	"github.com/sabhiram/go-gitignore"
 	"github.com/waigani/diffparser"
+	"github.com/zendesk/term-check/internal/config"
+	gh "github.com/zendesk/term-check/pkg/github"
```

Comment:
```suggestion
	 gh "github.com/zendesk/term-check/pkg/github"
```

Reply:
It doesn't look it, but it's actually in the same column as the `"`